examples/es6/src/index.ts

```diff
@@ -55,8 +55,18 @@ const returnOK = <T extends ResponseData>(res: Response, out: SdkResponse<T>) =>
 };
 
 const returnCookies = <T extends ResponseData>(res: Response, out: SdkResponse<T>) => {
-  if (out.ok && out.data?.cookies) {
-    res.set('Set-Cookie', out.data.cookies);
+  if (out.ok) {
+    // Set cookies with
+    // - Response's cookies
+    // - Response's session-token (if it exists)
+    // Note: Session token may grow, especially in cases of using authorization, or adding custom claims,
+    // This may cause it size to pass browser cookies size limit, use carefully.
+    const { cookies = [], sessionJwt = '' } = out?.data! || {};
+    const setCookies = [...cookies];
+    if (sessionJwt) {
+      setCookies.push(`${DescopeClient.SessionTokenCookieName}=${sessionJwt}`);
```

Comment:
Lets add the other values to the cookie
Secure
HttpOnly 
etc

Reply:
added, though I think its a bit problematic that this code is is required in the example

the sdk already has it in the `helpers.ts` file
![image](https://user-images.githubusercontent.com/10514677/208721620-326dbbaf-5154-4091-867e-83a860441ba5.png)

and I'm not sure that exposing it as an package API is the right thing to do

lets discuss separately